.circleci/config.yml

```diff
@@ -42,6 +42,20 @@ jobs:
     steps:
       - checkout
       - <<: *restore_dependency_cache
+      - run:
+          name: Install geckodriver
+          command: |
```

Comment:
Is it worth putting all this stuff in a `./.circleci/install-geckodriver.sh` file?